scripts/seed_data.py

```
@@ -0,0 +1,109 @@
+def create_entities(data):
+    # Create users
+    for user_data in data['users']:
+        _user, _created = User.objects.get_or_create(
+            uuid=user_data['uuid'],
+            defaults={
+                'username': user_data['username'],
+                'email': user_data['email'],
+                'first_name': user_data['first_name'],
+                'last_name': user_data['last_name'],
+                'role': user_data['role'],
+            }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)